src/storage/mvcc/txn.rs

```diff
@@ -355,6 +355,12 @@ mod tests {
         value
     }
 
+    fn maybe_write(engine: &Engine, ctx: &Context, modifies: Vec<Modify>) {
```

Comment:
maybe means the operation can be successful or not, but here the write must be ok if `modifiers` is not empty.